### PR TITLE
fix infinite alert when game is over

### PR DESCRIPTION
**Pull request by masquel**
Fix issue #1

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (PezCoder) [inline package-lock.json]: The game doesn't not using any external package, so we mightn't need this file.

File: snake.js [hunk 2]
```diff
@@ -287,9 +284,15 @@
       stopMotion({horizontal:false,length: -dy*4});
       y-=dy*5;
     }
-
+    if(lives === 0){
+      gameOver();
+    }
+  }
+  function gameOver(){
```
Comment (PezCoder) [inline snake.js]: Good idea for resetting the state, but there are more states in the game to keep track of & not just lives & score, Like the direction of the snake, no of snake blocks.

I think we can keep the reloading logic for now, but an `isGameOver`  boolean check with `lives === 0` should suffice.

File: snake.js [hunk 3]
```diff
@@ -287,9 +284,15 @@
       stopMotion({horizontal:false,length: -dy*4});
       y-=dy*5;
     }
-
+    if(lives === 0){
+      gameOver();
```
Comment (PezCoder) [inline snake.js]: Any specific reason for putting this inside `updateHead()` ? 
I think the purpose of updateHead should be to only update the positions of head & not anything else, It'll make it easier to track bugs later on & the code will be easier to understand.

The code related to game state should be belong in the main game loop i.e `draw()` function

**Review by PezCoder — CHANGES_REQUESTED** [review]
Thank you for the pr 👏🏼
I have a few suggestions.

File: package-lock.json (hunk 1 above)
Comment (masquel) [reply package-lock.json]: Yeah, sure. This was my habit to run `npm install` after cloning repo, so this file shouldn't be in repo.

File: snake.js (hunk 2 above)
Comment (masquel) [reply snake.js]: You can move all game data you need to reset after game is over to `gameData` object for example, for easy reset in future.

Comment (PezCoder) [reply snake.js]: Yeah, good idea.

**Comment by masquel** [comment]
@PezCoder I've moved check lives to `draw` function, added `resetGame` function in which you should reset all game data you need. All data i've reseted in this function is just for example. It's an idea how to solve your problem. Also i've remove package-lock.json

**Pull request closed without merging**